real_check/real_check.ps1

```diff
@@ -70,7 +70,14 @@ function Check-Symbols {
         return $return_code
 }
 
-exit (Check-Symbols -lib $args[0])
+ForEach($lib in $args){
+    Write-Host "Checking lib $lib"
+    $exit_code = Check-Symbols -lib $lib
```

Comment:
>$exit_code [](http://example.com/codeflow?start=4&length=10)

what's the exit code from `Check-Symbols` when it throws? I suppose line 45 could thrown some Not Enough Memory exception which doesn't seem to be caught.